01-retitle.Rmd

```diff
@@ -798,7 +798,7 @@ ggplot(data = dat,
 
 ### Cohort 1
 
-To be updated!
+`r knitr::include_url("https://youtu.be/lXWAfm0fh7Q")`
```

Comment:
This URL probably won’t work - I think it needs to be the URL from the embed code somehow, Jon and I tried this once

Reply:
(Pressing “approve” to build the book and see what happens)